packages/2019-template/src/components/TemplateAPICard/templateAPICardMeta.js

```diff
@@ -31,9 +31,7 @@ const TemplateAPICardMeta = (/* data */) => ({
   ),
   shareText:
     "Ice cream candy canes brownie marzipan jelly chocolate bar marshmallow.",
-  tags: [
-    /* "Transportation", "Bus", "Rail", "Portland" */
-  ],
+  tags: ["Transportation", "Bus", "Rail", "Portland", "Stuff"],
```

Comment:
This needs to stay commented out. We don't want cards that are a WIP to show up on the cards to go see.